src/Arcus.Observability.Telemetry.Serilog.Enrichers/Extensions/LoggerEnrichmentConfigurationExtensions.cs

```diff
@@ -44,5 +48,34 @@ public static LoggerConfiguration WithKubernetesInfo(this LoggerEnrichmentConfig
 
             return enrichmentConfiguration.With<KubernetesEnricher>();
         }
+
```

Comment:
With the model of extending CorrelationInfoEnricher by deriving from it we should give them the capability to pass it around here or use that one instead (I wish there was a `: new(<>)` :()

Reply:
It's not very much added value maybe (only `Enrich.With()` i's going to contain).